kbchat/kbchat_test.go

```diff
@@ -23,8 +25,19 @@ type Bots struct {
 	Charlie1 *OneshotOptions
 }
 
+type Team struct {
+	Teamname string
+	Channel  string
+}
+
+type Teams struct {
```

Comment:
this doesn't look like it needs to be in a type like this. `Acme` isn't really an attribute of a `Teams`.